test/Microsoft.ML.Benchmarks/Harness/Configs.cs

```diff
@@ -15,22 +15,21 @@ public RecommendedConfig()
 
             Add(GetJobDefinition() // job defines how many times given benchmark should be executed
                 .WithCustomBuildConfiguration(GetBuildConfigurationName())
-                .With(CreateToolchain())); // toolchain is responsible for generating, building and running dedicated executable per benchmark
+                .With(CreateToolchain())); // toolchain is responsible for generating, building and running dedicated executable per benchmark);
```

Comment:
Extraneous ');'

```suggestion
                .With(CreateToolchain())); // toolchain is responsible for generating, building and running dedicated executable per benchmark
```

Reply:
The number of `)` is fine, one for `CreateToolchain`, 2nd for `With` and 3rd for `Add` from line 16.

To prevent from such errors and breaking the benchmarks I added an integration test some time ago, so now if the build is green it means that we can run the benchmarks. https://github.com/dotnet/machinelearning/blob/19d907766a809bf0f91da15b5f10056dc2090942/test/Microsoft.ML.Benchmarks.Tests/BenchmarksTest.cs#L47

Reply:
I believe @justinormont was referring to the end of the comment like.  Comments in C# don't need to end in a semi-colon.  😝

Reply:
@justinormont I apology for the misunderstanding! 

@eerhardt  Thanks for pointing this out. It was hilarious ;p